kamon-akka-2.4.x/src/main/resources/reference.conf

```diff
@@ -58,4 +58,33 @@ kamon {
       requires-aspectj = yes
     }
   }
+}
+
+kanela {
+  show-banner = true
```

Comment:
Please remote show-banner and log-level.

Reply:
done